cmd/juju/caas/update.go

```diff
@@ -0,0 +1,406 @@
+// Copyright 2020 Canonical Ltd.
```

Comment:
There are still lots of code here are similar to `add.go`. 
But it seems that it's not very to refactor further.

Reply:
Yeah, there's common-ish code but the add command does querying of the k8s substrate to get storage info so it's different enough that it's messy to try and pull apart.